CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
 # CHANGELOG
 
+## 4.5.00
+
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.4.01...4.5.00)
+
+### Features
+
+* Introduce new `SequentialHostInit` view allocation property [\#7229](https://github.com/kokkos/kokkos/pull/7229) (backported in 4.4.01)
+* Support building with Run-Time Type Information (RTTI) disabled (missing PR)
```

Comment:
```suggestion
* Support building with Run-Time Type Information (RTTI) disabled
```
No PR to refer in particular.  This has been implemented over many.

Reply:
Ok, noted